src/RuleSet.php

```diff
@@ -155,6 +155,46 @@
             'psr4' => true,
             'silenced_deprecation_error' => true,
         ],
+        '@Laravel' => [
+            '@Symfony' => true,
```

Comment:
this is tricky unless `Laravel` decide to take `Symfony` CS guidelines with few additions. What I see, it is taking `PSR2` with few additions